ADAL/src/cache/ADTokenCacheAccessor.m

```diff
@@ -282,7 +283,13 @@ - (void)updateCacheToItem:(ADTokenCacheItem *)cacheItem
     NSString* savedRefreshToken = cacheItem.refreshToken;
     if (isMRRT)
     {
-        AD_LOG_VERBOSE(correlationId, @"Token cache store - Storing multi-resource refresh token for authority: %@", _authority);
+        if ([ADAuthorityUtils isKnownHost:[_authority adUrl]])
+        {
+          AD_LOG_VERBOSE(correlationId, @"Token cache store - Storing multi-resource refresh token for authority: %@", _authority);
```

Comment:
even if authority is a known host, we can only log host, because authority can contain tenant

Reply:
fixed